tests/module_tests.py

```diff
@@ -0,0 +1,28 @@
+import unittest
+
+import vimdoc
+from vimdoc.block import Block
+from vimdoc import error
+from vimdoc import module
+
+class TestVimPlugin(unittest.TestCase):
```

Comment:
Went with `TestVimModule` since `TestModule` sounded like a generic "this is a module of tests" to me.